cse-cc-import-runner-app/src/main/resources/application.yml

```diff
@@ -1,33 +1,55 @@
+server:
+  port: 8082
 spring:
   application:
-    name: cse-cc-runner
+    name: cse-idcc-runner
   rabbitmq:
-    host: localhost
+    host: 127.0.0.1
     port: 5672
     username: gridcapa
     password: gridcapa
     template:
       reply-timeout: 2400000
   cloud:
     stream:
+      function:
+        definition: interruptionConsumer;requestProcessor;notificationProducer
```

Comment:
This is surprising it works because it should be cloud.function.definition and not cloud.stream.function.definition

Reply:
I found it like this in some tutorial